FHIR3-0-2-MM202001-Dev/BgZ-3-0/_reference/resources-bgz-extended/medmij-bgz-ext-nl-core-patient-02.xml

```diff
@@ -41,15 +41,6 @@
         </given>
     </name>
     <telecom>
-	    <extension url="http://nictiz.nl/fhir/StructureDefinition/zib-ContactInformation-TelecomType">
-            <valueCodeableConcept>
-                <coding>
-                    <system value="http://hl7.org/fhir/v3/AddressUse"/>
-                    <code value="MC"/>
-                    <display value="Mobiel telefoonnummer"/>
-                </coding>
-            </valueCodeableConcept>
-        </extension>
         <system value="phone"/>
```

Comment:
![image](https://user-images.githubusercontent.com/20518398/98910868-09b55200-24c4-11eb-88d3-40f75af154b8.png)
 Deze combinatie staat mapping staat er niet echt tussen, toch?

Reply:
Alleen bij email hoeft er geen ext TelecomType te worden gebruikt volgens de mapping in nl-core-contactpoint. Klopt dat?

Reply:
@ArdonToonstra 

1. Die mapping staat er inderdaad niet bij, er is wel een example van overigens. Dit is een mobiel nummer waarvan je niet weet of hij werk/prive/tijdelijk is. Denk je dat we die moeten toevoegen? 
2. Dat we geen extensie bij email gebruiken is inderdaad zoals we het in MM-1165 hebben besloten met de Zomerrelease. 

@pieter-edelman-nictiz  wil jij hier ook nog even doorheen kijken voor de zekerheid?

Reply:
Gezien de hoeveelheid dat .use='mobile' in deze commit voorkomt zou ik hem wel toevoegen ja.